lib/release/lowlevel.rb

```diff
@@ -47,14 +47,39 @@ def submodule_status
       execute "git submodule status"
     end
 
+    # rubocop:disable Metrics/MethodLength
+    def retry_with_reset(max_attempts: 5, &block)
```

Comment:
I would put this method in the Tooling-module. I know the distinction is mostly made-up, but still.